### PR TITLE
Mark `flake8-2020` as complete

### PR DESCRIPTION
**Pull request by sobolevn**
Source: https://github.com/asottile/flake8-2020

### CODE REVIEW

**Comment by sobolevn** [comment]
This is really weird! GitHub is buggy right now.

File: stubs/flake8-2020/@tests/stubtest_allowlist.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# We don't need violation definitions:
+flake8_2020.YTT\d+
```
Comment (AlexWaygood) [inline stubs/flake8-2020/@tests/stubtest_allowlist.txt:2]: Why not?

File: stubs/flake8-2020/@tests/stubtest_allowlist.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
+# Has many implementation details:
+flake8_2020.Visitor
```
Comment (AlexWaygood) [inline stubs/flake8-2020/@tests/stubtest_allowlist.txt:5]: I don't think this actually has any public methods that aren't already included in typeshed's stub for the base class, `ast.NodeVisitor`: https://github.com/python/typeshed/blob/38d0f691e6afba8e88347726b7d56ed361ffd7bf/stdlib/ast.pyi#L37

**Review by AlexWaygood — APPROVED** [review]
Thanks!

**Comment by sobolevn** [comment]
@AlexWaygood shouldn't it be `def __getattr__(name: str) -> Incomplete: ...`?
I think it should. I will send a PR for this.

**Pull request merged**